### PR TITLE
use HanLP or Jieba create word_cloud_cn

### PR DESCRIPTION
**Pull request by TianFengshou**
pyhanlp is One of 
the most powerful natural language processing libraries in Chinese
today, and it's extremely easy to use.You can use 'PIP install pyhanlp'. 
To install it,like Jieba.

Its level of identity of named entity,word segmentation was better than jieba,
and has more ways to do it.You'll save a lot of time when you use it.

And thanks to its excellent performance,
when we handle a large number of Chinese texts
We don't have to use the User-defined dictionaries。

### CODE REVIEW

**Comment by jcfr** [comment]
Thanks for the contribution.

Would it be possible to squash the commits together and fix the styling issues ? 

To run the style check locally, you could run `flask8` from the source directory.

**Comment by FontTian** [comment]
What do I need to do now?

**Comment by FontTian** [comment]
What do I need to do now?

File: examples/wordcloud_cn.py [hunk 1]
```diff
@@ -63,11 +66,54 @@ def jieba_processing_txt(text):
     return ' '.join(mywordlist)
 
 
-wc = WordCloud(font_path=font_path, background_color="white", max_words=2000, mask=back_coloring,
-               max_font_size=100, random_state=42, width=1000, height=860, margin=2,)
+# The function for processing text with HaanLP
+def pyhanlp_processing_txt(text, isUseStopwordsOfHanLP=True):
+    CustomDictionary = JClass("com.hankcs.hanlp.dictionary.CustomDictionary")
```
Comment (amueller) [inline examples/wordcloud_cn.py]: JClass is not defined?

Comment (FontTian) [reply examples/wordcloud_cn.py]: If use hanlp must install jpype1 .I forgot about it......

"pip install jpype1"......

Comment (FontTian) [reply examples/wordcloud_cn.py]: Pyhanlp provides a python interface for hanlp. Hanlp is currently the best-performing open source Chinese natural language processing class library, but it is implemented in Java, so we must use jpype to call Java classes.

Comment (amueller) [reply examples/wordcloud_cn.py]: Ok but you must also import it, right?

Comment (amueller) [reply examples/wordcloud_cn.py]: You need to make sure the continuous integration passes.

Comment (FontTian) [reply examples/wordcloud_cn.py]: I don't need to import it.The pyhanlp import jpype1 in the __init__.py.

```python
from jpype import JClass, startJVM, getDefaultJVMPath, isThreadAttachedToJVM, attachThreadToJVM
```

Comment (FontTian) [reply examples/wordcloud_cn.py]: If use pyhanlp,In line 114 there's a line like “from pyhanlp import  *“
If use jieba,In line 106 there's a line like “import jieba“。This increases efficiency, and the code is feasible.

**Comment by amueller** [comment]
the tests still don't pass, meaning there are issues with your implementation. Are you intending to fix those?

**Comment by amueller** [comment]
Oh it's actually just flake8 failing now, it looks like.

**Comment by jcfr** [comment]
Indeed, error are the following:

```
./examples/wordcloud_cn.py:81: [F405] 'HanLP' may be undefined, or defined from star imports: pyhanlp
    HanLP.Config.ShowTermNature = False
    ^
./examples/wordcloud_cn.py:82: [F405] 'HanLP' may be undefined, or defined from star imports: pyhanlp
    CRFnewSegment = HanLP.newSegment("viterbi")
                    ^
./examples/wordcloud_cn.py:85: [E712] comparison to True should be 'if cond is True:' or 'if cond:'
    if isUseStopwordsByHanLP == True:
                             ^
./examples/wordcloud_cn.py:114: [F403] 'from pyhanlp import *' used; unable to detect undefined names
    from pyhanlp import *
    ^
./examples/wordcloud_cn.py:115: [F401] 'jpype.startJVM' imported but unused
    from jpype import JClass, startJVM, getDefaultJVMPath, isThreadAttachedToJVM, attachThreadToJVM
    ^
./examples/wordcloud_cn.py:115: [F401] 'jpype.getDefaultJVMPath' imported but unused
    from jpype import JClass, startJVM, getDefaultJVMPath, isThreadAttachedToJVM, attachThreadToJVM
    ^
./examples/wordcloud_cn.py:115: [F401] 'jpype.isThreadAttachedToJVM' imported but unused
    from jpype import JClass, startJVM, getDefaultJVMPath, isThreadAttachedToJVM, attachThreadToJVM
    ^
./examples/wordcloud_cn.py:115: [F401] 'jpype.attachThreadToJVM' imported but unused
    from jpype import JClass, startJVM, getDefaultJVMPath, isThreadAttachedToJVM, attachThreadToJVM
    ^
```

To run flake8 locally:

1. Activate your environment
2. Go to source directory
3. Make sure development requirements are installed 
4. Execute flake8

```
workon wordcloud # Or similar command to activate your python environment
cd /path/to/src/word_cloud
pip install requirements-dev.txt
flake8
```

**Comment by FontTian** [comment]
I downloaded flake8 and made some changes to the code. It may be ok now.

**Comment by FontTian** [comment]
With the development of time, Chinese natural language processing technology has become very mature. But jieba is still the fastest native python library, and other libraries need to call other codes or use deep learning models, so I personally think that as an example, maybe jieba is the best choice. As for the issue of optimal performance, it should be left to the user to solve. So it's almost time to close the pull.

**Comment by amueller** [comment]
So are you saying you'd leave the example as it was in #329 and close this pull request?

**Comment by TianFengshou** [comment]
I'm sorry for closing this merge request years later. I didn't merge it at the time due to concerns - it was just a demonstration, not an engineering application. Hanlp is still the best statistical based Chinese model to this day, and even has certain advantages over deep learning models in some fields. However, for a demonstration, his model is still too large.

**Pull request closed without merging**